app/helpers/courses_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# comment for StaticPages helper
```

Comment:
Be patient when you copy past something. Change comment and delete empty line 4. Run ./checkers